decidim-comments/app/events/decidim/comments/comment_created_event.rb

```diff
@@ -0,0 +1,56 @@
+# frozen-string_literal: true
+
+module Decidim
+  module Comments
+    class CommentCreatedEvent < Decidim::Events::BaseEvent
+      include Decidim::Events::EmailEvent
+      include Decidim::Events::NotificationEvent
+
+      def email_subject
+        I18n.t(
```

Comment:
Are all these methods using the recipient's locale?

Reply:
pinging @mrcasals since its delegated to the base event

Reply:
Checked it myself... yes, it is using the recipient's locale.